proc-macros/src/lib.rs

```diff
@@ -189,6 +191,14 @@ fn build_client_impl(api: &api_def::ApiDefinition) -> Result<proc_macro2::TokenS
 fn build_client_functions(api: &api_def::ApiDefinition) -> Result<Vec<proc_macro2::TokenStream>, syn::Error> {
 	let visibility = &api.visibility;
 
+	let _crate = match (crate_name("jsonrpsee-http-client"), crate_name("jsonrpsee-ws-client")) {
+		(Ok(FoundCrate::Name(name)), _) => syn::Ident::new(&name, Span::call_site()),
+		(_, Ok(FoundCrate::Name(name))) => syn::Ident::new(&name, Span::call_site()),
+		(_, Err(e)) => return Err(syn::Error::new(Span::call_site(), &e)),
```

Comment:
right, I think I need to support `FoundCrate::Itself` now because it should be possible to just use the one of the client crates and proc_macros ^^

Reply:
nvm I was wrong